apps/authentication-service/src/resources/clients/clients.routes.v1.ts

```diff
@@ -0,0 +1,86 @@
+import { clientCreateSchema } from "@/resources/clients/clients.validation";
```

Comment:
I think we've typically been using camel case for our file names. The exception has been example env files

Reply:
Are you suggesting I change all the files that follow this pattern? Happy to do it, just wanted to confirm.

Reply:
Also, should I change our e2e test files as well? Those follow a pattern of `testscope.cy.ts`.

Reply:
.spec .cy are about making a pattern than the test runners can find, so they should stay the same.

If we are just naming a file, then the name of the file should be before any "."

Yes I think we should change the files to follow this pattern.